### PR TITLE
Update web

### PR DESCRIPTION
**Pull request by ljacobson64**
* Reworked the main page as well as the install and tally docs
* Fixed formatting issues in the rest of the pages
* Reduced the sphinx build errors from 130+ to 4

I still think the main page needs work but this is a start...

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+    [       OK ] DagSolidTest.surface_area_test (5 ms)
+    [----------] 16 tests from DagSolidTest (228 ms total)
+
+    [----------] Global test environment tear-down
+    [==========] 16 tests from 1 test case ran. (228 ms total)
+    [  PASSED  ] 16 tests.
+
+If the tests have completed successfully, you are now ready to run your first
+DAGMC problem. See the `DAGMC workflow guides <workflow.html>`_ for more
```
Comment (makeclean) [inline doc/install.rst]: The associated workflow.rst file is not currently in your repo it seems

Comment (makeclean) [inline doc/install.rst]: It was deleted in fact, you should probably resurrect it

**Comment by ljacobson64** [comment]
@makeclean the file was still there but with a different name.

**Comment by ljacobson64** [comment]
Also I'm confused about what the purpose of the [dagmc_specific_options](https://github.com/svalinn/DAGMC/blob/update_web/doc/usersguide/workflow/dagmc_specific_options.rst) page is; there doesn't seem to be much useful information there

**Comment by makeclean** [comment]
Awesome, thanks a lot @ljacobson64 this looks great

**Pull request merged**